src/trello/views/attachments_view.py

```diff
@@ -0,0 +1,22 @@
+from django.shortcuts import render
+from rest_framework import generics
+from rest_framework.generics import get_object_or_404
+from trello.permissions import *
+from trello.models import *
+from rest_framework.permissions import IsAuthenticated, IsAdminUser
+from trello.serializers.attachments_serializers import *
+
+
+class AttachmentCreateView(generics.CreateAPIView):
+	serializer_class = AttachmentDetailSerializer
+	permission_classes = (IsAuthenticated, )
+
+	def perform_create(self, serializer):
+		serializer.save(task=self.request.task)
```

Comment:
This is not correct. In backend u receive request with data. This data go throw serializer and save send to obj.create() automatically.
For example if you want create attachment u just send post request to your endpoint wit data:
{
task: 12, //this is id of task
file: buffer_of_file,
...other fields
}